.github/CODEOWNERS

```diff
@@ -1,4 +1,9 @@
-* @Xuanwo @PsiACE @ClSlaid @sundy-li
+* @Xuanwo
 
+/bindings/c/ @Ji-Xinyou
+/bindings/haskell/ @silver-ymz
+/bindings/java/ @tisonkun
+/bindings/ruby/ @oowl
```

Comment:
oowl has been voted as an OpenDAL committer, but it will need time to become active.

Reply:
It should be lua here.